### ISSUE TITLE
Fee reimbursement

### ISSUE THREAD

**Issue by cbeams** [bisq-network/support#5]
@eespn commented on [Thu Jan 04 2018](https://github.com/bisq-network/exchange/issues/1148)

Hi, I am having this problem frequently, this is the forth time, I do not understand why you do not take a commission only after confirming the buy, I have already another ticket opened via email for the previous 3 cases.

![image](https://user-images.githubusercontent.com/35086537/34547848-edd68a60-f0b2-11e7-800e-d2960017b552.png)

I receive a message that there has been an error and a link to open a ticket in GitHUb or to open the log, now the only I can do is cancel, once I do that I get charged either way, it tells me that I have already funded the offer, that if I cancel the funds will be moved to the wallet again, but it didn't happen with the whole amount the last three times... so I have no hopes this time,

![image](https://user-images.githubusercontent.com/35086537/34547889-350c4b4a-f0b3-11e7-9201-51699f7e2a2f.png)

If I try t cancel and try to buy again it will not let me, the "Review: Take offer to buy Bitcoin" button is not functional any longer

![image](https://user-images.githubusercontent.com/35086537/34547924-73043de0-f0b3-11e7-8094-9a198ebd0c61.png)

This time, since it is the fourth time, I closed the tab without hitting cancel, hoping it will not charge me the deposit, and it worked (had to re-start Bisq for my balance to update), **BUT** still, a mysterious amount was withdrawn  "0.00205640", which is no small amount, it is over 30 USD

![image](https://user-images.githubusercontent.com/35086537/34548006-13a4916e-f0b4-11e7-90a2-f086ce1a9bcb.png)

As you can see from the image above, from the wallet an amount of 0.00205640 was withdraw, I can't understand why , can't make sense of it
 
Please assist to get this amount back, unfortunately have three additional cases before even worst than this one....,

Thanks!

---

@ManfredKarrer commented on [Thu Jan 04 2018](https://github.com/bisq-network/exchange/issues/1148#issuecomment-355189874)

Can you send me the log file (account/backup -> button)? My email is manfred at bitsquare dot io.

Please contact @cbeams regarding the lost fee payments.

---

@eespn commented on [Thu Jan 04 2018](https://github.com/bisq-network/exchange/issues/1148#issuecomment-355351027)

Thanks, I will be sending you the log soon after with code Ticket 1148 EESPN

**Comment by cbeams** [comment]
This transaction has been staged for reimbursement in commit e7d57dad63007b41a4f0af937879d3c1437722b0.

**Comment by cbeams** [comment]
Closing as complete. Reimbursement was paid out in transaction [9aa7f26d5b39301a0add1b627bb8e8cd961fb8d16c9777a3342ad160c6cf1ebf](https://tradeblock.com/bitcoin/tx/9aa7f26d5b39301a0add1b627bb8e8cd961fb8d16c9777a3342ad160c6cf1ebf). See also https://github.com/bisq-network/support/issues/1#issuecomment-358532451.
